### DISCUSSION TITLE
Port as a python library (written in Rust)

### DISCUSSION THREAD

**Discussion by expenses** [mxgmn/MarkovJunior#66] [Show and tell]
Hi, I actually started to write this a while ago, but I've written a Rust implementation of (a limited subset of) MarkovJunior as a python library: https://github.com/expenses/markov/. See
https://github.com/expenses/markov/blob/058f2ba26babcf07386ab6c79287dae2108397c0/examples/dungeon.py
and 
https://github.com/expenses/markov/blob/058f2ba26babcf07386ab6c79287dae2108397c0/examples/flowers.py

for examples of the nystrom dungeon generator and the flower field respectively.

**Comment by mxgmn** [comment]
Added a link!
